packages/plugin-core/src/test/suite-integ/NoteLookupCommand.test.ts

```diff
@@ -1120,7 +1120,14 @@ suite("NoteLookupCommand", function () {
         preSetupHook: async ({ wsRoot, vaults }) => {
           await ENGINE_HOOKS.setupBasic({ wsRoot, vaults });
         },
-        onInit: async ({ vaults, engine }) => {
+        onInit: async ({ wsRoot, vaults, engine }) => {
+          withConfig(
+            (config) => {
+              config.lookup.note.leaveTrace = true;
+              return config;
+            },
+            { wsRoot }
+          );
```

Comment:
can we leave the current test as-is and make a separate test case for `leaveTrace`?